### ISSUE TITLE
Basic Searching Algorithm Implementation In JavaScript

### ISSUE THREAD

**Issue by kuldeepsingh33** [kuldeepsingh33/DSA-Basic#38]

**Comment by Aritra056GITHUB** [comment]
I can help you with this

**Comment by deepesh85b** [comment]
@kuldeepsingh33 
Please assign this to me

**Comment by coderboy-yash** [comment]
Hey @kuldeepsingh33  can you please assign this issue to me under #hacktoberfest label
I can solve this issue

**Comment by Aritra056GITHUB** [comment]
ok I'll not let you down

**Comment by Aritra056GITHUB** [comment]
@kuldeepsingh33 I have made a pull request for basic sorting algorithms using javascript 
if (you find it good)
   please merge it 
else 
   I'll make the necessary changes.
Thank you! :)

**Comment by Aritra056GITHUB** [comment]
@kuldeepsingh33 you have merged my pl but hacktoberfest is not accepting it. Can you please see to that I am looking forward to get the 4/4 mark asap!

**Comment by kuldeepsingh33** [comment]
@Aritra056GITHUB  yes, due to some reason this repo was excluded from hacktoberfest for the contribution you can make a pull request 
here https://github.com/kuldeepsingh33/DSA-Hack

**Comment by Aritra056GITHUB** [comment]
@kuldeepsingh33 ok sure post the issue of sorting I'll put the pull request asap.
